docs/getting_started.md

```diff
@@ -51,23 +51,20 @@ If you are not interested in backing up any data, the `--gcs.bucket` flag can si
 
 Thanos comes with a highly efficient gRPC-based Store API for metric data access across all its components. The sidecar implements it in front of its connected Prometheus server. While it is ready to use with the above example, we must additionally configure the sidecar to join a Thanos cluster.
 
-Components in a Thanos cluster are connected through a gossip protocol to advertise membership and propagate metadata about the data and services they provide.
-
-We a set of initial peers addresses to the sidecar. They only must be valid on startup and subsequent cluster changes are automatically adapted to.
+Let's extend the above sidecar to expose their gRPC Store API so that we can query metrics.
```

Comment:
Can we add at line 49 something like: For details and configuration of other object stores see `./docs/storage.md`